src/components/input-icon-toggle/input-icon-toggle.component.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Icon from '../icon/icon';
+import './input-icon-toggle.scss';
+
+/**
+ * An InputIconToggle Component.
+ *
+ * == How to use an InputIconToggle in a component:
+ *
+ * In your file
+ *
+ *   import InputIconToggle from 'carbon-react/lib/components/input-icon-toggle';
+ *
+ * To render an InputIconToggle:
+ *
+ *   <InputIconToggle content='' iconType='foo' inputId='bar' />
+ */
+const InputIconToggle = (props) => {
+  const { iconType, content, inputId } = props;
+
+  return (
+    <label htmlFor={ inputId } key='label-icon'>
+      { content || <Icon type={ iconType } className='input-icon-toggle' /> }
+    </label>
+  );
+};
+
+InputIconToggle.propTypes = {
+  /**
```

Comment:
I think we're trying to minimise this documentation, so that it only contains the long description. The type, prop name and other attributes are readable from the `PropTypes` config.

Reply:
documentation trimmed.